### DISCUSSION TITLE
Python Lang

### DISCUSSION THREAD

**Discussion by kokonut27** [whippingdot/Language-Tutorials#7] [General]
hahahahaha, i actually did smth! lmao yay!

**Comment by whippingdot** [comment]
??

**Reply by darkdarcool** [reply]
lol nice jb

**Reply by whippingdot** [reply]
bruh y u replying to my comment for that? You can create ur own comment

**Reply by darkdarcool** [reply]
oh, whoops. srry @whippingdot
